scripts/03a-integrate-sce.R

```diff
@@ -252,8 +261,19 @@ if (integration_method == "fastmnn") {
     batch_column = opt$batch_column,
     cosine_norm  = opt$fastmnn_no_cosine,
     gene_list    = fastmnn_gene_list,
-    seed         = opt$seed
+    seed         = opt$seed,
+    auto.merge   = opt$fastmnn_auto_merge
```

Comment:
I'm going to respond to other comments, but I just wanted to note that the auto merge is being implemented in `fastmnn` here. Because we have allowed for additional arguments already using `...` in `integrate_fastMNN()`, I'm using the `auto.merge` argument here without modifying the underlying scripts. I can do that if we think that's necessary, but I just wanted to note that the auto merge is being enabled here. 

https://github.com/AlexsLemonade/sc-data-integration/blob/2376f30227a7f57f288b5bc2fc90b0ca556398f7/scripts/utils/integrate-fastmnn.R#L30-L35

Reply:
This is my preference too, since it leaves less room for error when updating in the future, which is why I didn't do it in the first place. I'm going to leave it out for now.

Reply:
> my preference would be to not to add more that we need to explicitly translate from one argument name to another.

makes sense!